meinberlin/apps/contrib/templates/meinberlin_contrib/item_detail.html

```diff
@@ -253,7 +253,6 @@ <h3 class="detail-info__section-title">{% translate 'Remark' %}</h3>
         {% endif %}
     {% endblock %}
         <section>
-            <h2 class="visually-hidden">{% translate 'Comments' %}</h2>
```

Comment:
already comes from the react comments

Reply:
I think we do need it in a+, so it's a bit of a clash. I guess we could remove it in a4 and then a+ has to set it in the templates where the comments are used. Would that be better?

Reply:
I updated the a4 PR to remove it

Reply:
@hom3mad3 just realized you mentioned "Feedback", that's not part of the comments, that's the moderator feedback. So we probably still want the `Comments` landmark? However, the "Join the Discussion" then should be a h3?

Reply:
I updated it in a4, should be better now